marin/speedrun/paloma_local_download.py

```diff
@@ -0,0 +1,73 @@
+"""
+The Paloma eval sets, downloaded and tokenized
+
+https://huggingface.co/datasets/allenai/paloma
+"""
+
+import os.path
+
+from marin.download import HfDownloadConfig
+from marin.download.huggingface.download_hf import download_hf
+from marin.execution.executor import ExecutorStep, executor_main, this_output_path, versioned
+from marin.processing.tokenize import TokenizeConfig
+from marin.processing.tokenize.data_configs import TokenizerStep
+
+llama3_tokenizer = "meta-llama/Meta-Llama-3.1-8B"
+
+
+# The datasets in the Paloma eval set and their paths within the HF dataset
+# https://huggingface.co/datasets/allenai/paloma
+PALOMA_DATASETS_TO_DIR = {
```

Comment:
Can we factor this out so we don't have to replicate it between local and non-local?